whirlpool/washerdryer.py

```diff
@@ -91,6 +146,51 @@ def get_machine_state(self):
                 return k
         return None
 
+    def get_machine_state_pretty(self):
+        machine_state = self.get_machine_state()
+        machine_state_raw = self.get_attribute(ATTR_MACHINE_STATE)
+        machine_state_pretty = ""
+        try:
+            machine_state_pretty = MACHINE_STATE_MAP_PRETTY[machine_state]
+        except:
+            return None
+
+        # Fix Maytag Machine bug -- Time remaining shows 0 (cycle complete) but state continues to be "Running Post Cycle"
+        if machine_state_raw == ATTRVAL_MACHINE_STATE_RUNNINGPOSTCYCLE:
+            print (machine_state_raw, ATTRVAL_MACHINE_STATE_RUNNINGPOSTCYCLE)
+            if self.get_time_remaining_mins() == 0:
+                return MACHINE_STATE_MAP_PRETTY[MachineState.Complete]
+        return machine_state_pretty
+
+
+    def get_time_remaining_mins(self):
+        return self.attr_value_secs_to_mins_int(self.get_attribute(ATTR_CYCLE_TIME_REMAINING))
+
+    #Handles delay time
+    def get_delay_time_remaining_mins(self):
+        try: 
+            return self.attr_value_secs_to_mins_int(self.get_attribute(ATTR_DELAY_TIME_REMAINING))
+        except:
+            return 0
+
+    def get_total_delay_and_cycle_time_remaining_mins(self):
+        delay_time_remaining = self.get_delay_time_remaining_mins()
+        cycle_time_remaining = self.get_time_remaining_mins()
+        return delay_time_remaining + cycle_time_remaining
```

Comment:
```suggestion
```
The sum would also be done in an upper layer like HA. This washerdryer class aims to provide direct values for most things, only converting types (int, bool, float) and enumerations to python enums, but it should not have any result manipulation.